provision/vipmanager.bash

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+declare -a ETCD
+
+while getopts "n:v:e:i:" option; do
+    case "${option}" in
+        n) NODENAME="${OPTARG}" ;;
+	i) VIP_ADDR="${OPTARG}" ;;
+        v) VIP_MANAGER_VERSION="${OPTARG}" ;;
+        e) ETCD[${#ETCD[@]}]=$OPTARG ;;
+    esac
+done
+
+if rpm -q vip-manager &>/dev/null ; then
+    echo "vipmanager was already installed"
+else
+    echo "Installing vipmanager"
+    rpm --import "/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7"
+    yum install -y --quiet wget
+
+    VIPMANAGER_RPM="vip-manager_${VIP_MANAGER_VERSION}-1_amd64.rpm"
```

Comment:
This should help getting the packag name without having to mess with version all over the place:

~~~
wget -qO- 'https://api.github.com/repos/cybertec-postgresql/vip-manager/releases/latest'|jq '.assets | .[] | select(.content_type == "application/x-rpm") | .browser_download_url'
~~~

Reply:
done that way.